.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "browser": true,
+    "webextensions": true
+  },
+  "plugins": ["svelte3", "unicorn"],
+  "extends": ["xo/browser", "plugin:unicorn/recommended"],
+  "rules": {
+    "indent": ["error", 2],
+    "import/no-unassigned-import": "off",
+    "no-multiple-empty-lines": [
+      "error",
+      {
+        "max": 1,
+        "maxBOF": 2,
+        "maxEOF": 1
+      }
+    ],
```

Comment:
Why modify this rule, instead of sticking to original max-1-empty-line rule?

Reply:
https://github.com/sveltejs/eslint-plugin-svelte3/issues/41#issuecomment-572503966

I commented the code now